packages/client/ansible-collection-theforeman-foreman/ansible-collection-theforeman-foreman.spec

```diff
@@ -0,0 +1,57 @@
+%global debug_package %{nil}
+
+%global collection_namespace theforeman
+%global collection_name foreman
+%global collection_directory %{_datadir}/ansible/collections/ansible_collections/%{collection_namespace}/%{collection_name}
+
+%if 0%{?fedora} || 0%{?rhel} >= 8
+%global ansible_python python3
+%global pyyaml %{ansible_python}-pyyaml
+%else
+%global ansible_python python2
+%global pyyaml PyYAML
+%endif
+
+%global release 1
+
+Name:       ansible-collection-%{collection_namespace}-%{collection_name}
+Version:    0.1.0
+Release:    %{?prerelease:0.}%{release}%{?prerelease}%{?nightly}%{?dist}
+Summary:    The Foreman Project Ansible modules collection
+
+License:    GPLv3+
+URL:        https://theforeman.org/plugins/foreman-ansible-modules
+Source0:    https://galaxy.ansible.com/download/%{collection_namespace}-%{collection_name}-%{version}.tar.gz
+BuildArch:  noarch
+
+Requires: ansible >= 2.8
+Requires: %{ansible_python}-apypie >= 0.0.5
+Requires: %{pyyaml}
+%if 0%{?rhel} == 7
+Requires: python-ipaddress
+%endif
+
+%description
+Collection of Ansible Modules to manage Foreman installations.
+Includes modules for Katello.
+
+%prep
+%setup -q -c
+
+%build
+
+%install
+mkdir -p %{buildroot}%{collection_directory}
+cp -a ./* %{buildroot}%{collection_directory}
```

Comment:
I'm still more a fan of the explicit paths. Looking at what it now copies is also `tests/sanity/ignore-2.10.txt` which doesn't appear all that useful. Are you doing this so `FILES.json` is correct?

Reply:
I am doing this to match what `ansible-galaxy collection install` would do, and it puts *everything* into that folder.